Parser.fs

```diff
@@ -386,13 +405,13 @@ let parseParam : Parser<Param, unit> =
     pipe2ws parseType parseIdentifier buildParam
     // ^ <type> <identifier>
 
-
 (*
  * Views.
  *)
 
 /// Parses a conditional view.
 let parseIfView =
+    // TODO: use parseIflike.
```

Comment:
This TODO is optional, really.  Changing to this would make view conditionals look like

`{| if (x) { y } else { z} |}`

which is ugly but consistent.